packages/angular/src/builders/module-federation-dev-server/schema.json

```diff
@@ -122,6 +122,13 @@
     "pathToManifestFile": {
       "type": "string",
       "description": "Path to a Module Federation manifest file (e.g. `my/path/to/module-federation.manifest.json`) containing the dynamic remote applications relative to the workspace root."
+    },
+    "moduleFederationConfigOptions": {
+      "type": "object",
+      "description": "Custom Module Federation configuration to be used instead of the default configuration generated by the builder. This can be useful for multi-repository module federation setups where the host application uses a remote application from an external repository.",
+      "properties": {
+        "path": { "type": "string" }
+      }
```

Comment:
We should simplify this to just be of type string

```json
"moduleFederationConfig": {
  "type": "string",
  "description": "...",
}
```

Reply:
It might also be worth noting that if this supports paths outside of the project root, that the inputs for caching for this project will also need to be updated to include the path to this file. 

This warning should potentially be added to the description.

If the project's inputs are not updated to include this file (if it lives outside the project root), then you can encounter cache hits even when you change this config file.